staging/src/k8s.io/kube-gen/cmd/conversion-gen/generators/conversion.go

```diff
@@ -338,36 +338,42 @@ func (e equalMemoryTypes) Equal(a, b *types.Type) bool {
 	if equal, ok := e[conversionPair{b, a}]; ok {
 		return equal
 	}
-	result := e.equal(a, b)
+	result := e.equal(a, b, alreadyVisitedTypes)
```

Comment:
Wondering, do we need the parameter in`Equal` at all? Can't we create the empty map here?